src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  height: 100vh;
+
+  @include flex-center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 10px;
+  gap: $main-gap;
+  width: ($main-size * 7) + ($main-gap * 6);
+  background-color: #fff;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $main-size;
+    width: $main-size;
+
+    background-color: $day-bg-color;
+    border: 1px solid #000;
+    transition: background-color transform 0.5s;
```

Comment:
The `transform` property should be included in the transition property list only if you are transitioning it. If not, remove it to avoid any confusion or unexpected behavior.